src/lib/functions/registry.ts

```diff
@@ -350,7 +348,6 @@ export class FunctionsRegistry {
       const icon = warningsText ? NETLIFYDEVWARN : NETLIFYDEVLOG
       const color = warningsText ? chalk.yellow : chalk.green
       const mode =
-        // @ts-expect-error TS(2339) FIXME: Property 'logLambdaCompat' does not exist on type ... Remove this comment to see the full error message
         func?.runtimeAPIVersion === 1 && this.logLambdaCompat
```

Comment:
This was actually a legit bug: `logLambdaCompat` was a static field so this was always falsy.